demos-europe/demosplan-core
# feat(refs DPLAN-12526): add statement quicksave button after ocr text recognition

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-12526/3-2-Als-Importerin-mochte-ich-meine-Anpassungen-nach-der-OCR-Texterkennung-zwischenspeichern-konnen-um-sicherzugehen-dass-meine


This PR adds the possibility to quick save the annotated statement text so adjustments can be persisted to be used at a later time without needing to finish this step completely, which could result in a loss of the work.

### Linked PRs (optional)

PR Backend: https://github.com/demos-europe/demosplan-addon-demospipes/pull/247

### How to test:
- install this demospipes version: https://github.com/demos-europe/demosplan-addon-demospipes/tree/f-DPLAN-12526-add-quicksave-property
- checkout branch
- run migration `bin/ewm doc:mig:migrate --configuration=addons/cache/demosplan-addon-demospipes-f-DPLAN-12526-add-quicksave-property/config/addon_migrations.yml`
- after ocr text recognition a new button should be visible under the editor component and the text should be savable now (resource type gets an update request)

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
